tests/AutoReview/CiConfigurationTest.php

```diff
@@ -257,4 +258,20 @@ private function getPhpVersionsUsedByGitHub(): array
 
         return $phpVersions;
     }
+
+    /**
+     * @return list<numeric-string>
+     */
+    private function getPhpVersionsUsedByGitHubDocker(): array
+    {
+        $yaml = Yaml::parse(file_get_contents(__DIR__.'/../../.github/workflows/docker.yml'));
+
+        $phpVersions = [];
+
+        foreach ($yaml['jobs']['build-and-push-image']['strategy']['matrix']['include'] as $job) {
+            $phpVersions[] = $job['php-version'];
+        }
+
+        return $phpVersions;
```

Comment:
return array_map(
    item => item['php-version']
    $yaml['jobs']['build-and-push-image']['strategy']['matrix']['include']
);

Reply:
Changed